packages/dd-trace/src/instrumenter.js

```diff
@@ -74,14 +74,19 @@ class Instrumenter {
 
   enable (config) {
     config = config || {}
+    const serviceMapping = config.serviceMapping
 
     this._enabled = true
 
     if (config.plugins !== false) {
       Object.keys(plugins)
         .filter(name => !this._plugins.has(plugins[name]))
         .forEach(name => {
-          this._set(plugins[name], { name, config: getConfig(name) })
+          const config = {}
+          if (serviceMapping && serviceMapping[name]) {
+            config.service = serviceMapping[name]
```

Comment:
Environment variables should have less priority than code.

Reply:
They do. If someone does `tracer.use(...)` after this, that will win. Or are you referring to something else?

Reply:
Ah yes it does, sorry I missed that. Can you add this environment variable to the config priority test?